packages/cloud_firestore/cloud_firestore/example/test_driver/load_bundle_e2e.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:typed_data';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:cloud_firestore_platform_interface/cloud_firestore_platform_interface.dart';
+import 'package:http/http.dart' as http;
+
+void runLoadBundleTests() {
```

Comment:
A test for error cases would be lovely (converting platformexceptions and not converting non-platform exceptions)